dependencyLibs/Core/src/org/droidplanner/core/MAVLink/command/doCmd/MavLinkDoCmds.java

```diff
@@ -99,6 +103,40 @@ public static void setServo(Drone drone, int channel, int pwm) {
     }
 
     /**
+     * Set the mount_mode of the gimbal.  This decides how the gimbal decides where to point.
+     * Should it listen to RC input, should it point at an ROI point, or should it point to
+     * a MAVLink direction.
+     *
+     * @param drone
+     * @param mountMode What mode should the gimbal be in?  See MAV_MOUNT_MODE
+     * @param stabilizePitch Leave these false unless the gimbal is a servo gimbal.
+     * @param stabilizeRoll
+     * @param stabilizeYaw
+     * @return
+     */
+    public static void configureGimbal(Drone drone, int mountMode, boolean stabilizePitch,
+                                       boolean stabilizeRoll, boolean stabilizeYaw) {
+        if (drone == null)
+            return;
+
+        Parameter mountParam = drone.getParameters().getParameter("MNT_MODE");
+        if(mountParam==null){
+            msg_mount_configure msg = new msg_mount_configure();
+            msg.target_system = drone.getSysid();
+            msg.target_component = drone.getCompid();
+            msg.mount_mode = (byte) mountMode;
+            msg.stab_pitch = stabilizePitch ? (byte) 1  : 0;
+            msg.stab_roll = stabilizeRoll ? (byte) 1 : 0;
+            msg.stab_yaw = stabilizeYaw ? (byte) 1 : 0;
+            drone.getMavClient().sendMavPacket(msg.pack());
+        }else{
+            drone.getParameters().sendParameter("MNT_MODE", 1, mountMode);
+            mountParam.value = mountMode;
```

Comment:
This is the only part that I'm not sure about.  I needed to send a new parameter.  This was the best I came up with.  Is there a better way?